.github/workflows/release-please.yml

```
@@ -0,0 +1,24 @@
+  
+  permissions:
+    contents: write
+    pull-requests: write
+  
+  name: release-please
+  
+  jobs:
+    release-please:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: googleapis/release-please-action@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)